velox/connectors/hive/storage_adapters/gcs/RegisterGcsFileSystem.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include "velox/connectors/hive/storage_adapters/gcs/RegisterGcsFileSystem.h"
```

Comment:
nit: We don't need to include this header.

Reply:
Same as abfs, it's needed for the function signature. If it's not included then the compilation will fall with `VELOX_ENABLE_GCS` undefined.